crates/blockifier/src/execution/syscalls/syscall_tests/secp.rs

```diff
@@ -3,31 +3,46 @@ use test_case::test_case;
 
 use crate::abi::abi_utils::selector_from_name;
 use crate::context::ChainInfo;
-use crate::execution::call_info::CallExecution;
+use crate::execution::call_info::{CallExecution, CallInfo};
 use crate::execution::entry_point::CallEntryPoint;
 use crate::execution::native::utils::NATIVE_GAS_PLACEHOLDER;
 use crate::test_utils::contracts::FeatureContract;
 use crate::test_utils::initial_test_state::test_state;
 use crate::test_utils::{trivial_external_entry_point_new, CairoVersion, BALANCE};
 
-// TODO: Native
-// #[test_case(FeatureContract::SierraTestContract, NATIVE_GAS_PLACEHOLDER; "Native")]
-#[test_case(FeatureContract::TestContract(CairoVersion::Cairo1), 17035610; "VM")]
```

Comment:
We should keep the original test attributes to be inline with how the tests are written